week_1/project/week_1.py

```diff
@@ -50,16 +51,25 @@ def get_s3_data(context):
     return output
 
 
-@op
-def process_data():
-    pass
+@op(
+    ins={"stock_list": In(dagster_type=List[Stock])},
+    out={"agg": Out(dagster_type=Aggregation)},
+    description="Find the highest high value from a list of stocks"
+    )
+def process_data(stock_list):
```

Comment:
I don't know if it aids the IDE to add
```
process_data(stock_list: list[Stock]) -> Aggregation:
```
or if that is just redundant In/Out typing information.

Reply:
🤔, Hrm, PyCharm doesn't seem to care. Could be explicitness.

Reply:
I'm not sure if the `-> Aggregation` piece does anything operationally. From my reading, it just adds annotation to the function so that if someone is looking in the docs for the expected output type as `Aggregation`, it would be there.